src/client.ts

```diff
@@ -77,6 +79,7 @@ export class KeycloakAdminClient {
     this.clientScopes = new ClientScopes(this);
     this.clientPolicies = new ClientPolicies(this);
     this.identityProviders = new IdentityProviders(this);
+		this.clientRegistrationPolicies = new ClientRegistrationPolicies(this);
```

Comment:
Same here, make sure to indent with spaces.
```suggestion
    this.clientRegistrationPolicies = new ClientRegistrationPolicies(this);
```